qe_metrics/cli.py

```diff
@@ -0,0 +1,70 @@
+import os
+from typing import Dict, Any
+
+import click
+
+from simple_logger.logger import get_logger
+
+from qe_metrics.libs.database import Database
+from qe_metrics.libs.jira import Jira
+
+
+@click.command()
+@click.option(
+    "--services-file",
+    default=os.environ.get("QE_METRICS_SERVICES", "services.yaml"),
+    help="Defines the path to the services file.",
```

Comment:
what the service file contains? I can't understand it from the help.

Reply:
I will add some context, in the help. This is the file that will define the services and their queries. [Here is the example file](https://github.com/RedHatQE/qe-metrics/blob/97328ec100d3f29d052bfba6e3de8e23db9296f7/qe_metrics/manifests/services.example.yaml). We can also name it something else if you feel there is a better name.